actionpack/lib/action_dispatch/middleware/exception_wrapper.rb

```diff
@@ -174,6 +174,29 @@ def self.status_code_for_exception(class_name)
       Rack::Utils.status_code(@@rescue_responses[class_name])
     end
 
+    def show?(request)
+      # We're treating `nil` as "unset", and we want the default setting to be
+      # `:all`. This logic should be extracted to `env_config` and calculated
```

Comment:
Explore if we can really move this logic to `env_config`.

Reply:
Previous exploration: https://github.com/rails/rails/pull/46609#issuecomment-1330659207